packages/client/src/internal/conversions/modernToLegacyWhereClause.ts

```diff
@@ -15,8 +15,7 @@
  */
 
 import type {
-  GeoFilter_Intersects,
```

Comment:
Cannot remove these two types from api package because that would be a break

Reply:
So why can we remove them?